### DISCUSSION TITLE
--- stderr   thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ffigen is not a command, or not executable.', build.rs:99:56

### DISCUSSION THREAD

**Discussion by A10ha** [rustdesk/rustdesk#1431] [General]
--- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ffigen is not a command, or not executable.', build.rs:99:56
How do I solve？

**Comment by btc2100k** [comment]
I'm having this issue too, not sure where to even begin to diagnose this.

**Comment by btc2100k** [comment]
To anyone running into this issue, I fixed it by pulling the latest release branch (in my case, `git checkout 1.1.9`), and then ran cargo run, and everything built as expected.

**Reply by A10ha** [reply]
Thanks, I'll try it！

**Comment by kajack** [comment]
The error is because the flutter sdk is missing from the path.
Also ran  dart pub global activate ffigen --version 5.0.1

**Reply by kajack** [reply]
Forgot to mention stuck at another mistake.

**Reply by fufesou** [reply]
`flutter pub global activate ffigen` also works.
